pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+markers =
```

Comment:
Whenever I specify the markers I like using the `--strict-markers` too:

```
  --strict-markers, --strict
                        markers not registered in the `markers` section of the
                        configuration file raise errors.
```

That will prevent typos by not recognizing markers that are not registered. Also this ensure that the markers list will be keep up to date.